src/murmur/MumbleServer.ice

```diff
@@ -117,7 +117,7 @@ module MumbleServer
 		IntList links;
 		/** Description of channel. Shown as tooltip for this channel. */
 		string description;
-		/** Channel is temporary, and will be removed when the last user leaves it. */
+		/** Channel is temporary, and will be removed when the last user leaves it. Read-only. */
```

Comment:
I'm not sure Read-only sufficiently explains the behavior.

```suggestion
		/** Channel is temporary, and will be removed when the last user leaves it. Read-only; can not currently be set on existing nor new channel. */
```

It doesn't make sense when creating channels because they need at least one user. Alternatively, we could support creating empty temporary channels, and they disappear after someone joined and then the last user left.

For editing, in the current behavior, a user would have to be inside of it to be settable, so it does not instantly disappear.

All in all, it may be simplest to leave it as it is: Not supporting temporary via Ice API.

Reply:
I think read-only captures the gist rather well

Reply:
@Krzmbrzl Did you want to accept this MR as-is then?

Reply:
Yes

Reply:
Please mark it as approved then